src/gapstate.h

```diff
@@ -99,8 +99,8 @@ typedef struct GAPState {
 
     /* From code.c */
     Stat * PtrBody;
-    Stat   OffsBody;
-    Stat * OffsBodyStack;
+    UInt   OffsBody;
+    UInt * OffsBodyStack;
```

Comment:
Fair point. I am not entirely happy with how `Stat` and `Exec` are used, though: As you write, on the one hand they are simply offsets. On the other hand, we pretend that the body of a function is a sequence of `Stat`, which of course is not correct: There are also StatHeaders in there, and various other encoded data (e.g. variable ids). Also, the distinction between `Stat` and `Expr` is completely artificial. It can of course be useful in code, to make clear what the intended content of some variable is, but since the types are identical, it's also trivial to mix them up. 

That said, in the end, I see no good solution for this. I'll drop the `OffsBody` changes from this PR now, though, as I don't think it's a good idea to have some `Stat` replaced by `UInt` and others not... I'll think some more about it

(Perhaps `StatOffset` and `ExprOffset` would be slightly better names, but in the end, it doesn't really help either *shrug*

Reply:
(also, this is complaining on a very nitpicky level. All in all, the stat/expr encoding is quite nice, IMHO